src/components/SelectClaims/SelectClaims.tsx

```diff
@@ -18,7 +18,7 @@ type SelectOption = {
 type Props = RouteComponentProps<{}> & {
   closeMenuOnSelect?: boolean
   claims?: Claims.Entry[]
-  cTypeHash: ICType['cType']['hash'] | null
+  cTypeHash: ICType['ctypeHash'] | null
```

Comment:
`ICType` here is coming from `'../../types/Ctype'`. are you sure about this change?

Reply:
No, I removed it. I guess when I did some changes it auto-suggested this.

Reply:
You changed `ICType` coming from the demo-client to `ICType` coming from the sdk, which is ok, but unnecessary for this PR.

Reply:
I have reverted it back to its original way.